### PR TITLE
error on headerString[key].split(2,/:/)

### PR DESCRIPTION
**Pull request by marcosmoski**
the last argument of array is a function, you can not do split on function
i have added if(typeof headerString[key] != 'function') to fix it

### CODE REVIEW

File: lib/ofx.js [hunk 1]
```diff
@@ -161,7 +161,9 @@ OFX.parse = function (ofxStr, fn) {
   data.header = {};
 
   for(var key in headerString){
-    var headAttributes = headerString[key].split(/:/,2);
+    if (typeof headerString[key] != "function"){
```
Comment (euforic) [inline lib/ofx.js]: I think it would be better to test if it is a string

Comment (karissa) [inline lib/ofx.js]: also you probably want to use `!==`

Comment (marcosmoski) [inline lib/ofx.js]: I agree with you it's better teste if it is a string, and now i use the "===".

**Comment by euforic** [comment]
Please squash you commits or remove the first incorrect one

**Pull request merged**